python/mergeTreeClustering.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+
+from paraview.simple import *
+
+# create a new 'XML Image Data Reader'
+isabelvti = XMLImageDataReader(registrationName='isabel.vti', FileName=['isabel.vti'])
+isabelvti.PointArrayStatus = ['velocityMag_02', 'velocityMag_03', 'velocityMag_04', 'velocityMag_05', 'velocityMag_30', 'velocityMag_31', 'velocityMag_32', 'velocityMag_33', 'velocityMag_45', 'velocityMag_46', 'velocityMag_47', 'velocityMag_48']
+isabelvti.TimeArray = 'None'
+
+all_JT_group = []
+all_ST_group = []
+
+for scalarField in isabelvti.PointArrayStatus:
+  # create a new 'TTK Merge and Contour Tree (FTM)'
+  tTKMergeandContourTreeFTM1 = TTKMergeandContourTreeFTM(Input=isabelvti)
+  tTKMergeandContourTreeFTM1.ScalarField = ['POINTS', scalarField]
+  tTKMergeandContourTreeFTM1.InputOffsetField = ['POINTS', scalarField]
```

Comment:
I don't believe this is needed for the script to work.

Reply:
Indeed! Done in the next commit!